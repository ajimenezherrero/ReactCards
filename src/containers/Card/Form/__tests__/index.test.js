import React from 'react';
import { shallow } from 'enzyme';

import CardsForm from '../index';

const mockComponent = props => <CardsForm { ...props }/>;

describe('<CardsForm />', () => {
  it('should render component', () => {
    const tree = shallow(mockComponent());

    expect(tree).toMatchSnapshot();
  });
});
