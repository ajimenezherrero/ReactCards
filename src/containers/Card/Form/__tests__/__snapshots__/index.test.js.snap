// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardsForm /> should render component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CardsForm />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <styled.form
        onSubmit={[Function]}
      >
        <label>
          Title:
          <input
            name="title"
            onChange={[Function]}
            type="text"
            value=""
          />
        </label>
        <label>
          Description:
          <input
            name="description"
            onChange={[Function]}
            type="text"
            value=""
          />
        </label>
        <label>
          Image(url):
          <input
            name="image"
            onChange={[Function]}
            type="text"
            value=""
          />
        </label>
        <input
          type="submit"
          value="Submit"
        />
      </styled.form>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <label>
            Title:
            <input
              name="title"
              onChange={[Function]}
              type="text"
              value=""
            />
          </label>,
          <label>
            Description:
            <input
              name="description"
              onChange={[Function]}
              type="text"
              value=""
            />
          </label>,
          <label>
            Image(url):
            <input
              name="image"
              onChange={[Function]}
              type="text"
              value=""
            />
          </label>,
          <input
            type="submit"
            value="Submit"
          />,
        ],
        "onSubmit": [Function],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Title:",
              <input
                name="title"
                onChange={[Function]}
                type="text"
                value=""
              />,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Title:",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "name": "title",
                "onChange": [Function],
                "type": "text",
                "value": "",
              },
              "ref": null,
              "rendered": null,
              "type": "input",
            },
          ],
          "type": "label",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Description:",
              <input
                name="description"
                onChange={[Function]}
                type="text"
                value=""
              />,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Description:",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "name": "description",
                "onChange": [Function],
                "type": "text",
                "value": "",
              },
              "ref": null,
              "rendered": null,
              "type": "input",
            },
          ],
          "type": "label",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Image(url):",
              <input
                name="image"
                onChange={[Function]}
                type="text"
                value=""
              />,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Image(url):",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "name": "image",
                "onChange": [Function],
                "type": "text",
                "value": "",
              },
              "ref": null,
              "rendered": null,
              "type": "input",
            },
          ],
          "type": "label",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "type": "submit",
            "value": "Submit",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
      ],
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <styled.form
          onSubmit={[Function]}
        >
          <label>
            Title:
            <input
              name="title"
              onChange={[Function]}
              type="text"
              value=""
            />
          </label>
          <label>
            Description:
            <input
              name="description"
              onChange={[Function]}
              type="text"
              value=""
            />
          </label>
          <label>
            Image(url):
            <input
              name="image"
              onChange={[Function]}
              type="text"
              value=""
            />
          </label>
          <input
            type="submit"
            value="Submit"
          />
        </styled.form>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <label>
              Title:
              <input
                name="title"
                onChange={[Function]}
                type="text"
                value=""
              />
            </label>,
            <label>
              Description:
              <input
                name="description"
                onChange={[Function]}
                type="text"
                value=""
              />
            </label>,
            <label>
              Image(url):
              <input
                name="image"
                onChange={[Function]}
                type="text"
                value=""
              />
            </label>,
            <input
              type="submit"
              value="Submit"
            />,
          ],
          "onSubmit": [Function],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Title:",
                <input
                  name="title"
                  onChange={[Function]}
                  type="text"
                  value=""
                />,
              ],
            },
            "ref": null,
            "rendered": Array [
              "Title:",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "name": "title",
                  "onChange": [Function],
                  "type": "text",
                  "value": "",
                },
                "ref": null,
                "rendered": null,
                "type": "input",
              },
            ],
            "type": "label",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Description:",
                <input
                  name="description"
                  onChange={[Function]}
                  type="text"
                  value=""
                />,
              ],
            },
            "ref": null,
            "rendered": Array [
              "Description:",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "name": "description",
                  "onChange": [Function],
                  "type": "text",
                  "value": "",
                },
                "ref": null,
                "rendered": null,
                "type": "input",
              },
            ],
            "type": "label",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Image(url):",
                <input
                  name="image"
                  onChange={[Function]}
                  type="text"
                  value=""
                />,
              ],
            },
            "ref": null,
            "rendered": Array [
              "Image(url):",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "name": "image",
                  "onChange": [Function],
                  "type": "text",
                  "value": "",
                },
                "ref": null,
                "rendered": null,
                "type": "input",
              },
            ],
            "type": "label",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "type": "submit",
              "value": "Submit",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
        ],
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
